scss/ticketInfo.scss

```diff
@@ -0,0 +1,20 @@
+@import "./variables.scss";
```

Comment:
No need for @import variables since they are imported in main.scss and ticketInfo is importet there.

Reply:
cool, I didnt know that. Thank you so much. That would explain why my sass war adding three links 🤣